src/routes/JobSummary/components/cells/ItemCell.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import {
+  getRecordActionStatusLabel,
+  getHotlinkCellFormatter,
+  BaseLineCell,
+} from '../utils';
+
+import { RECORD_ACTION_STATUS } from '../../../../utils';
+
+export const ItemCell = ({
+  sortedItemData,
+  instanceId,
+}) => {
+  const itemsInfoCell = sortedItemData?.map((sortedItems, itemIndex) => {
+    const groupOfItems = sortedItems?.map((item, index) => {
+      const entityLabel = getRecordActionStatusLabel(item.actionStatus);
+      const holdingsId = item.holdingsId;
+      const itemId = item.id;
+      const path = `/inventory/view/${instanceId}/${holdingsId}/${itemId}`;
+      const isDiscarded = item.actionStatus === 'DISCARDED';
```

Comment:
@OleksandrHladchenko1 
```suggestion
      const isDiscarded = item.actionStatus === RECORD_ACTION_STATUS.DISCARDED;
```